ApiBundle/Transformer/NodeGroupRoleTransformer.php

```diff
@@ -52,25 +66,43 @@ public function transform($nodeGroupRole)
     }
 
     /**
-     * @param FacadeInterface             $facade
-     * @param NodeGroupRoleInterface|null $source
+     * @param FacadeInterface     $nodeRoleFacade
+     * @param GroupInterface|null $group
      *
      * @throws RoleNotFoundHttpException
-     * @return mixed
+     * @throws TransformerParameterTypeException
+     *
+     * @return null|NodeGroupRoleInterface
      */
-    public function reverseTransform(FacadeInterface $facade, $source = null)
+    public function reverseTransform(FacadeInterface $nodeRoleFacade, $group = null)
```

Comment:
I agree with @itkg-alavieille : an integrator developping a new class must respect the interface, our presenting only 2 params. If we should look at the class, the interface is useless or in our case non operationnal.

Plus there's a great problem with https://github.com/open-orchestra/open-orchestra-cms-bundle/blob/master/ApiBundle/Transformer/AreaTransformer.php#L223 : if the $node is trully null, it severly crashes. I'd be for a refacto of this part.
A basic solution should be to put a third mixed param to allow each class to pass an optional parameter.